data/core/encyclopedia/calendar.cfg

```diff
@@ -11,10 +11,8 @@
 [topic]
     id=..calendar
     title= _ "Calendar"
-    text= _ "Each year in the Wesnoth calendar is composed of 12 months. These are, in order:
+    text= _ "Each year in the Jessenian calendar, names after Lady jessene, is composed of 12 months, starting around early spring and ending in late winter. These are, in order:
```

Comment:
Even if we decide to make this canonical - and I'm not sure we should - I suggest 'named for Lady Jessene'.

Reply:
I’ve changed it accordingly. Personally I think this sound better than “Wesnoth Calendar”, but it might be bestt to just call it Wesnothian.

Reply:
There seems to be a lack of "why is this part of the worldbuilding" and "what does this add for the player".

If it's named after someone or something that suggests it was created around the time of TRoW, then there should be some reason why the calendar of the Green Isle wasn't still used, even though Wesnoth's part of the Great Continent has the same length of year and probably similar seasons. Maybe the Wesfolk and Loyalists had different calendars, which would suggest names taken from both, yet the names don't seem to suggest different cultures.

However, if we don't have a reason to add that, I think it should be a case of "less is more". If there's paragraphs of text about something, it should be because story within the mainline campaigns ties into that.

Reply:
Yeah, I hadn't gone so far as to consider _why_ it was named after any particular individual, merely pointing out that it _could have been_. I like the name "Jessenian", but I can see why you'd think it hard to justify it in lore.